app/src/main/java/ch/epfl/sdp/entity/ShelterArea.java

```diff
@@ -29,7 +30,7 @@ public GeoPoint getLocation() {
 
     @Override
     public void displayOn(MapApi mapApi) {
-        mapApi.displayMarkerCircle(this, 0x7fffbf, "Shelter Area", 100);
+        mapApi.displayMarkerCircle(this, 0x7fffbf, "Shelter Area", (int) this.aoeRadius);
```

Comment:
A possible refactor we can do is to change all the aoeRadius (and a lot of variable) to int, because one meter is nothing in the real world, especially knowing the location finders